Simplenote/Classes/SPNoteEditorViewController.m

```diff
@@ -934,29 +959,32 @@ - (void)endSearching:(id)sender {
 #pragma mark Keyboard Notifications
 
 - (void)keyboardWillChangeFrame:(NSNotification *)notification {
+    
     NSDictionary *userInfo  = notification.userInfo;
     NSTimeInterval duration = [((NSNumber *)userInfo[UIKeyboardAnimationDurationUserInfoKey]) floatValue];
     CGRect keyboardFrame    = [((NSValue *)userInfo[UIKeyboardFrameEndUserInfoKey]) CGRectValue];
     CGSize viewBounds       = self.view.bounds.size;
 
-    CGFloat visibleHeight   = MAX(viewBounds.height - CGRectGetMinY(keyboardFrame), 0);
-    BOOL isEditing          = visibleHeight > 0;
+    CGFloat newKeyboardHeight = MAX(viewBounds.height - CGRectGetMinY(keyboardFrame), 0);
+    BOOL isKeyboardVisible = newKeyboardHeight > 0;
     
     void (^animations)() = ^void() {
         CGRect newFrame            = self->_noteEditorTextView.frame;
-        newFrame.size.height       = self.view.frame.size.height - (self->bVoiceoverEnabled ? self->_tagView.frame.size.height : 0) - visibleHeight;
-        if (!isEditing) {
+        newFrame.size.height       = self.view.frame.size.height - (self->bVoiceoverEnabled ? self->_tagView.frame.size.height : 0) - newKeyboardHeight;
+        if (!isKeyboardVisible) {
             newFrame.size.height -= self.view.safeAreaInsets.bottom;
         }
         self->_noteEditorTextView.frame  = newFrame;
         
         if (self->bVoiceoverEnabled) {
             CGRect newFrame        = self->_tagView.frame;
-            newFrame.origin.y      = self.view.frame.size.height - self->_tagView.frame.size.height - visibleHeight;
+            newFrame.origin.y      = self.view.frame.size.height - self->_tagView.frame.size.height - newKeyboardHeight;
             self->_tagView.frame         = newFrame;
         }
     };
     
+    // TODO: Drop isInteractive code path.
+    // Should only be necessary on pre-iOS 9 devices.
```

Comment:
**Actually** this was related to the (old) custom transition between List <> Editor. But should definitely be reviewed / dropped